### ISSUE TITLE
Cannot create encrypted config via external script

### ISSUE THREAD

**Issue by thecount2a** [ncw/rclone#3127]
#### What is the problem you are having with rclone?

Rclone cannot currently create a new encrypted config, when being invoked via external script (for example, with stdin or using environment variables)

#### What is your rclone version (output from `rclone version`)

rclone v1.47.0
- os/arch: linux/amd64
- go version: go1.12.4

#### Which OS you are using and how many bits (eg Windows 7, 64 bit)

Linux Fedora 25, 64bit

####  Which cloud storage system are you using? (eg Google Drive)

Google Drive

I have discussed this in a thread in the forum:

https://forum.rclone.org/t/headless-token-refresh-or-headless-setup-of-encrypted-configuration/9677

@ncw suggested an approach that I think would solve this quite well:

> RCLONE_CONFIG_PASS=password rclone --config /tmp/config config update newremote type drive

Basically, if a password is defined via the environment, even if no password is required to decrypt the current config, the changes should be made and the config should be written back out encrypted with the password which is defined via the environment.

**Comment by ncw** [comment]
Would someone like to have a go at this?  I don't think it is very complicated?

**Comment by czosnek16** [comment]
I want to do it.

**Comment by ncw** [comment]
> I want to do it.

Great. 

What needs to happen is this bit of code

https://github.com/ncw/rclone/blob/f0e439de0df77b6ceee197d62f51b47455fe8a3a/fs/config/config.go#L278-L285

Needs to run when the config file is not found

https://github.com/ncw/rclone/blob/f0e439de0df77b6ceee197d62f51b47455fe8a3a/fs/config/config.go#L216-L222

then I think it should all work as `configKey` will be set when we come to save the config.

Perhaps all the loading the key from the environment should be moved up in the file a bit.
